src/main/java/net/minecraftforge/common/ForgeModContainer.java

```diff
@@ -352,13 +352,14 @@ else if (VERSION_CHECK_CAT.equals(event.getConfigID()))
     }
 
     @Subscribe
-    public void missingMapping(FMLMissingMappingsEvent event){
-        for (FMLMissingMappingsEvent.MissingMapping evt:event.getAll())
+    public void missingMapping(FMLMissingMappingsEvent event)
+    {
+        for (FMLMissingMappingsEvent.MissingMapping entry : event.getAll())
         {
-            if (evt.name.equals("minecraft:totem"))//This item changed from 1.11 -> 1.11.2
+            if (entry.name.equals("minecraft:totem")) //This item changed from 1.11 -> 1.11.2
             {
                 ResourceLocation newTotem = new ResourceLocation("minecraft:totem_of_undying");
-                evt.remap(Item.REGISTRY.getObject(newTotem));
+                entry.remap(Item.REGISTRY.getObject(newTotem));
```

Comment:
Since this is serving as an example for modders, it should definitely use `ForgeRegistries`.